config/core.extension.yml

```diff
@@ -46,5 +46,6 @@ theme:
   classy: 0
   bartik: 0
   seven: 0
+profile: config_installer
```

Comment:
This is required in 8.3 due to changes made in https://www.drupal.org/commitlog/commit/2/40f803ac9dd3e36926d5236e74e01e48a03fa51a specifically, [this line](http://cgit.drupalcode.org/drupal/tree/core/lib/Drupal/Core/EventSubscriber/ConfigImportSubscriber.php?id=40f803ac9dd3e36926d5236e74e01e48a03fa51a#n138) requires a profile to be set so it's guaranteed the post-install profile matches the pre-install (config) profile.

For this reason we can't use "standard" or anything other than "config_installer".